lib/libzfs/libzfs_util.c

```diff
@@ -474,6 +478,10 @@ zfs_standard_error_fmt(libzfs_handle_t *hdl, int error, const char *fmt, ...)
 	case ZFS_ERR_WRONG_PARENT:
 		zfs_verror(hdl, EZFS_WRONG_PARENT, fmt, ap);
 		break;
+	case ZFS_ERR_BADPROP:
+		zfs_error_aux(hdl, strerror(error));
```

Comment:
This doesn't look right, `error` here will be `ZFS_ERR_BADPROP` which won't map to any standard `strerror()`.  Perhaps just drop the `zfs_error_aux()` call, unless you had something specific in mind.

Reply:
Fixed